docs/yarn.lock

```diff
@@ -154,11 +154,24 @@
     "@babel/highlight" "^7.22.13"
     chalk "^2.4.2"
 
+"@babel/code-frame@^7.23.5":
```

Comment:
can you check **Docs CloudFlare Deploy / deploy (push)**  should be passing. 
if this lock file is generated by you we can merge it, but if it was generated by any external bot? I'll suggest to generate by your self. Thanks

Reply:
Done
Thanks,